Chat app css illustration

Daniel4lm opened the pull request:
## Subject

This pull request contains the solution to the Chat app CSS illustration challenge on Frontend Mentor, 
as a part of my Optimum BH Internship. 

The task was to build out feature illustration for simple Booking App. The right section is atext details section. The left one is a mobile interface design that shows the chat pattern interface with messages.
Users can see the chat interface animation on the initial load.

### The solution on this branch contains the following:

- [x] Html page containing the corresponding DOM structure, including the Optimum header with anchor tags with urls(whose presense was requested by script tests).
- [x] Tailwind configuration file containing the extended theme atributes, like colors, sizes, breakpoints, etc.
- [x] Challenge starter files, incl. images and style guide
- [x] Solution design images(mobile & desktop layout)
- [x] CSS style file with tailwind directives and custom classes

## *Note*

After finishing the test, created screenshots of both versions(mobile & desktop) are not shown properly because of animation inside CSS code.

Daniel4lm committed:
Implementation of Chat App CSS ilustration

- Iclude challenge starter files
- Implement HTML DOM structure
- Tailwind theming & additional configuration

Daniel4lm committed:
Some inetersting things

- Fix some graphic bugs
- Animate chat interface messages on initial page load

Daniel4lm committed:
- Modify README.md
- Add result screenshot images

almirsarajcic commented on challenges/chat_app_css_illustration/style-guide.md:
```diff
@@ -0,0 +1,46 @@
+# Front-end Style Guide
```
Remove this file.

Daniel4lm replied:
Done

almirsarajcic reviewed (changes requested):
The mobile variant is not scrollable. Besides that, great work 👍

Daniel4lm committed:
Review update

- Remove README.md & style-guide.md files
- Credits are now visible in both layouts
- Mobile version is scrollable

Daniel4lm commented:
> The mobile variant is not scrollable. Besides that, great work +1

I think it was actually scrollable on mobile version, but anyway let me know if the issue exist.

Daniel4lm committed:
Fix source code link
